install/kubernetes/helm/istio/README.md

```diff
@@ -116,6 +116,11 @@ The chart deploys pods that consume minimum resources as specified in the resour
     EOF
     ```
 
+1. Add `istio.io` chart repository and point to the release:
+    ```
+    $ helm repo add istio.io https://storage.googleapis.com/istio-prerelease/daily-build/master-latest-daily/charts
```

Comment:
Since this is release-1.1, you probably want to use https://storage.googleapis.com/istio-prerelease/daily-build/release-1.1-latest-daily/charts instead.

Reply:
the code in `release-1.1` will be merged back to `master`. so if change it to release-1.1, it is confused in `master`. so how about just keep as it is? the released packages have the correct url.

Reply:
@hklai Does it make sense? Thanks.

Reply:
The same is true for many other things, like api and proxy shas, and the versions hardcoded in some helm config.

I suppose we should keep this specific to release-1.1 as it might have (or will be) diverged from master.

Reply:
okay. let us use `https://storage.googleapis.com/istio-prerelease/daily-build/release-1.1-latest-daily/charts` instead.